NodeBase/start-selenium-node.sh

```diff
@@ -31,6 +31,10 @@ if [ ! -z "$SE_OPTS" ]; then
   echo "Appending Selenium options: ${SE_OPTS}"
 fi
 
+if [ ! -z "$SE_DRIVER_OPTS" ]; then
+  echo "Appending Selenium Driver Service property: ${SE_DRIVER_OPTS}"
+fi
+
```

Comment:
Please revert this change, we always want to set the browser driver property to avoid Selenium Manager in these images.